### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Drdoteam**
Hi.

In JAX-RS section, adding 'JAXRSContract' is omitted.
So, I didn't understand the JAX-RS example correctly. And I got an exception when I run it.
If adding 'JAXRSContract' is added, It is helpful to beginners like me.

Thank you.

### CODE REVIEW

**Comment by adriancole** [comment]
Looks good,  but let's try to keep the fole as short as possible. I think lines 149-152 are the most essential to add. Do you agree?

**Comment by Drdoteam** [comment]
Yes, I do.
It looks much better to understand.

File: README.md [hunk 1]
```diff
@@ -144,6 +144,8 @@ interface GitHub {
   @GET @Path("/repos/{owner}/{repo}/contributors")
   List<Contributor> contributors(@PathParam("owner") String owner, @PathParam("repo") String repo);
 }
+...
```
Comment (adriancole) [inline README.md]: Tip If you append java to the triple backquote, it will format nicely.

Comment (adriancole) [inline README.md]: Last nit: I would still retain newlines after the call to each builder. Some mobile viewers will see an awkward wrap otherwise.

Comment (Drdoteam) [inline README.md]: Did you mean this ? Two java code blocks and retaining newlines.
I need to study English more. T.T 

```java
interface GitHub {
  @GET @Path("/repos/{owner}/{repo}/contributors")
  List<Contributor> contributors(@PathParam("owner") String owner, @PathParam("repo") String repo);
}
```
```java
GitHub github = Feign.builder()
                     .contract(new JAXRSModule.JAXRSContract())
                     .target(GitHub.class, "https://api.github.com");           
```

**Comment by adriancole** [comment]
Thanks!

**Comment by Drdoteam** [comment]
You're welcome ~ 
I was happy to contribute to the feign.
I'm using it for making some API clients in my company. And I want to contribute continuously!

**Pull request merged**